CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## x.y.z
 
 ### Added
+- Run the CI build once a week
+  ([#932](https://github.com/MyIntervals/emogrifier/pull/932),
```

Comment:
The URL should contain `issues` rather than `pull` - if needed at all (I think often but not always, historically, we have just linked the PR, as that should be sufficient).

Reply:
Thank. Changed, rebased and repushed.